ch02.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[ch02_getting_started]]
+== Getting Started
+
+// TODO intro paragraph - no empty headings!
+
+==== Choosing a Lightning Network Wallet
+
+* full nodes (c-lightning, eclair, lnd) + remote controls
+* phone / desktop wallets (SPV clients)
+* custodial services / wallets?
+// Mastering bitcoin also had a section about custodial web wallets. So it might be fair to include them.
+
+==== Quick Start
+
+* We start with a mobile wallet like eclair
+* Verify the authenticity of the software (e.g. App store / Acinq website / github)
+** discuss that a general phising scheme might consist of tricking you to download a similar looking software
+
+[[getting_first_bitcoin]]
+==== Getting Your First Bitcoin on the Lightning Network
+
+* Trade fiat for Bitcoin (as in Mastering Bitcoin)
```

Comment:
Will this be onchain to eclair and then open the channel yourself or will the other person open the channel with you and push liquidity? I would suggest the second option (as we can't go for turbo channels yet)

Reply:
I'm guessing just but Bitcoin elsewhere, and send it to your wallet of choice.